provider/icu4x-datagen/src/main.rs

```diff
@@ -476,23 +460,19 @@ fn main() -> eyre::Result<()> {
                 .map(|family_str| family_str.parse().wrap_err(family_str))
                 .collect::<eyre::Result<Vec<_>>>()?,
         };
-        let mut options = FallbackOptions::default();
-        options.deduplication_strategy = match cli.deduplication {
-            Some(Deduplication::Maximal) => Some(icu_datagen::DeduplicationStrategy::Maximal),
-            Some(Deduplication::RetainBaseLanguages) => {
-                Some(icu_datagen::DeduplicationStrategy::RetainBaseLanguages)
-            }
-            Some(Deduplication::None) | None => Some(icu_datagen::DeduplicationStrategy::None),
-        };
-        options.runtime_fallback_location = match cli.runtime_fallback_location {
-            Some(RuntimeFallbackLocation::Internal) => {
-                Some(icu_datagen::RuntimeFallbackLocation::Internal)
-            }
-            Some(RuntimeFallbackLocation::External) => {
-                Some(icu_datagen::RuntimeFallbackLocation::External)
-            }
-            None => Some(icu_datagen::RuntimeFallbackLocation::External),
+        let mut options = match cli.format {
+            Format::Dir | Format::Blob | Format::Blob2 => FallbackOptions::no_deduplication(),
+            Format::Mod => FallbackOptions::maximal_deduplication(),
```

Comment:
This is fine because it defers the question about what to make the baked default, but we may want to use `retain_base_languages` here.

Suggestion: move the `TODO(2.0): Default to RetainBaseLanguages here` down here (ideally link to an actual issue number)